bool subscribe
bool[8] adcChannels
uint16 adcFreqAcq
uint16 adcFreqPoll
uint32 adcBufferSize
---
int8 ret
string errorDesc